go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/google/go-cmp v0.5.2 // indirect
 	github.com/google/go-github/v30 v30.0.0
 	github.com/goreleaser/goreleaser v0.129.0
+	github.com/hashicorp/go-retryablehttp v0.6.7
```

Comment:
Unfortunately, the MPL-2.0 licensed is not [approved by the CNCF](https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md) and [this repo does not have an exemption](https://github.com/cncf/foundation/blob/master/license-exceptions/cncf-exceptions-2019-11-01.json). We could ask the CNCF for an exemption as they have for other hashicorp repos. Should we ask?

Reply:
it is being used in other projects: https://github.com/kubernetes/minikube/blob/master/deploy/minikube/release_sanity_test.go#L28

the search query: https://cs.k8s.io/?q=github.com%2Fhashicorp%2Fgo-retryablehttp&i=nope&files=&excludeFiles=&repos=

Reply:
Not sure if it's worth it. How long would that take? I could just implement the retry myself instead.

Reply:
@cpanato Interesting. Looks like we'd be in good company.

Reply:
I decided to use a different approach for the time being. I figured we already use another retry lib which I am now using.